pkgs/by-name/re/re-isearch/package.nix

```
@@ -47,14 +50,22 @@ stdenv.mkDerivation {
   preInstall = ''
     mkdir -p $out/{bin,lib}
   '';
-  postInstall = ''
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/{bin,lib}
+
+    cp ../bin/{Iindex,Isearch,Iutil,Idelete,zpresent,Iwatch,zipper} $out/bin
     cp ../lib/*.so $out/lib/
+
+    runHook postInstall
   '';
 
-  meta = with lib; {
+  meta = {
     description = "Novel multimodal search and retrieval engine";
     homepage = "https://github.com/re-Isearch/";
-    license = licenses.asl20;
-    maintainers = [ maintainers.astro ];
+    license = lib.licenses.asl20;
```

Let's specify that this only builds on `x86_64-linux`.

```suggestion
    license = lib.licenses.asl20;
    platforms = [ "x86_64-linux" ];
```

It never worked for `aarch64-linux` anyways, but we can investigate making it work in a followup PR.

---

Sorry for the late Commit, Been a bit busy past 2 days.

---

No worries. Life happens.